kasmvnc/README.md

```diff
@@ -29,11 +31,24 @@ Automatically install [KasmVNC](https://kasmweb.com/kasmvnc) in a workspace, and
    module "kasmvnc" {
      source              = "https://registry.coder.com/modules/kasmvnc"
      agent_id            = coder_agent.example.id
-     version             = "1.0.0"
+     custom_version      = "1.0.0"
```

Comment:
just curious: why are you calling it custom_version?

Reply:
version is a reserved keyword and is not allowed in terraform.